Locale/ruRU/config.lua

```diff
@@ -227,15 +234,7 @@ QuestieLocale.locale['ruRU'] = {
     ['MAP_GLOBAL_SCALE_DESC'] = "Размер значков карты.  ( По умолчанию: %s )",
     ['MINIMAP_GLOBAL_SCALE'] = "Глобальная шкала для иконок мини-карт",
     ['MINIMAP_GLOBAL_SCALE_DESC'] = "Размер значков мини-карты. ( По умолчанию: %s )",
-    ['ENABLE_OBJECTIVES'] = "Включить значки целей",
-    ['ENABLE_OBJECTIVES_DESC'] = "Когда включено, значки задач квеста будут отображаться на карте/миникарте",
-    ['ENABLE_TURNINS'] = "Включить значки завершенных квестов",
-    ['ENABLE_TURNINS_DESC'] = "Когда включено, места сдачи квестов будут отображаться на карте/миникарте",
-    ['ENABLE_AVAILABLE'] = "Включить доступные значки квестов",
-    ['ENABLE_AVAILABLE_DESC'] = "Когда включено, доступные квестовые локации будут отображаться на карте/миникарте",
-    ['ENABLE_TOOLTIPS'] = "Включить подсказки",
-    ['ENABLE_TOOLTIPS_DESC'] = "Когда включено, информация о квесте будет добавлена ??в соответствующие подсказки моба / предмета.",
-    ['MINIMAP_ALWAYS_GLOW_TOGGLE'] = "Подсветка иконок миникарты",
+	['MINIMAP_ALWAYS_GLOW_TOGGLE'] = "Подсветка иконок миникарты",
```

Comment:
Could you remove the additional tab on this?
```suggestion
    ['MINIMAP_ALWAYS_GLOW_TOGGLE'] = "Подсветка иконок миникарты",
```

Reply:
done